### PR TITLE
Fix about link

### LINKED ISSUE #334: The Help>>About menu item should work on Ubuntu
Clicking on Help>>About on Ubuntu does nothing. It should instead open up the Galleon site (https://cryptonomic.tech/galleon.html) in the browser.

### PR DESCRIPTION
**Pull request by dariuszsobkowicz**
Close #334

### CODE REVIEW

**Review by vishakh — APPROVED** [review]

**Review by vishakh — APPROVED** [review]

File: src/config.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "tagline": "containers.loginHome.tagline",
     "LocalVersionIndex": "34",
     "helpUrl": "https://github.com/Cryptonomic/T2",
+    "aboutUrl": "https://cryptonomic.tech/galleon.html",
```
Comment (anonymoussprocket) [inline src/config.json]: "T2" builds aren't "Galleon" builds.

**Review by anonymoussprocket — COMMENTED** [review]
Please see comment

File: src/config.json (hunk 1 above)
Comment (vishakh) [reply src/config.json]: @dariuszsobkowicz This should be read from a config file.

**Comment by dariuszsobkowicz** [comment]
@anonymoussprocket @vishakh url updated

**Review by anonymoussprocket — APPROVED** [review]

**Pull request merged**